README.md

```diff
@@ -150,7 +150,13 @@ Or use latest pre-built image from [DockerHub](https://hub.docker.com/r/qdrant/q
 docker pull qdrant/qdrant
 ```
 
-To run container use command:
+To run the container, use the command:
+
+```bash
+docker run -p 6333:6333 qdrant/qdrant
+```
+
+And once you need a fine-grained setup, you can also define a storage path and custom configuration:
```

Comment:
How about adding a short note on persistence?
"Please note that this won't persist your data if the container is restarted. And once you need..."

Reply:
actually, restarting container in this scenario will work fine. But re-creating the container will drop data